roles/oc_setup/templates/99_ssh_mc.yml.j2

```
@@ -0,0 +1,22 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: {{ os_role }}
+  name: 99-{{ os_role }}-ssh
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    passwd:
+      users:
+      - name: core
+        sshAuthorizedKeys:
+  {% for ssh_key_path in oc_ssh_extra_keys_paths %}
+      - '{{ lookup('file', ssh_key_path ) }}'
+  {% endfor %}
```

Is the `~` expanded as expected? iirc, with ansible is not possible, I think it's needed to be replaced with the `HOME` env var, e.g.

```suggestion
  {% for ssh_key_path in oc_ssh_extra_keys_paths %}
      - '{{ lookup('file', ssh_key_path | regex_replace('~', lookup('env', 'HOME'))) }}'
  {% endfor %}